.github/workflows/deploy_docs.yml

```diff
@@ -26,21 +26,34 @@ jobs:
         working-directory: ${{ env.docs-directory }}
         run: |
           make html
-      - name: Archive documentation
-        uses: actions/upload-artifact@v4
-        with:
-          name: documentation.tar.gz
-          path: ${{ env.docs-directory }}/generated_docs
-      # .nojekyll file is needed for GitHub Pages to know it's getting a ready webpage
```

Comment:
hmm did you accidentally delete half the comment?

Reply:
Fixed.